src/main/java/com/github/ibm/mapepire/ws/DbWebsocketClient.java

```diff
@@ -60,30 +57,35 @@ private static DataStreamProcessor getDataStream(DbWebsocketClient endpoint, Sys
     InputStream in = new ByteArrayInputStream(new byte[0]);
 
     OutputStream outStream = new OutputStream() {
-      private StringBuilder string = new StringBuilder();
+      private final List<Byte> payload = new ArrayList<>();
```

Comment:
Can you replace this with a `ByteArrayOutputStream`? This would eliminate the need for the loop currently down in `getBytes()`.